src/Polly.SharedSpecs/Registry/DefaultPolicyRegistrySpecs.cs

```diff
@@ -0,0 +1,238 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Xunit;
+using FluentAssertions;
+using Polly.Registry;
+using Polly.NoOp;
+
+namespace Polly.SharedSpecs.Registry
+{
+    public class DefaultPolicyRegistrySpecs
+    {
+        IPolicyRegistry<string, Policy> _registry;
+
+        public DefaultPolicyRegistrySpecs()
+        {
+            _registry = new DefaultPolicyRegistry();
+        }
+
+        [Fact]
+        public void Should_Allow_Adding_Policy_Using_Add()
```

Comment:
Done. Didn't knew about this. I also tend to keep things consistent and follow the practice of the project. Will keep this in mind in my future contributions. 😁